README.md

```diff
@@ -1,2 +1,8 @@
 # docker-yamllint
-yamllint packaged as a container image, published by Contane
+
+[yamllint](https://github.com/adrienverge/yamllint) packaged as a container
+image, published by [Contane](https://contane.net).
+
+The image is Alpine-based. By default, it runs in the `/data` directory with
+the root user. A non-root user named `yamllint` is also available and should
+be used if possible.
```

Comment:
I think we should add usage instructions here as well.

Reply:
Done